mm2src/coins/qrc20.rs

```diff
@@ -1367,6 +1367,7 @@ impl MmCoin for Qrc20Coin {
         &self,
         value: TradePreimageValue,
         stage: FeeApproxStage,
+        include_refund_fee: bool,
     ) -> TradePreimageResult<TradeFee> {
         let decimals = self.utxo.decimals;
         // pass the dummy params
```

Comment:
There is a code repetition related to dummy params in `get_sender_trade_fee` function and in `get_receiver_trade_fee`

![Screenshot 2024-04-09 at 17 25 37](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/3b99ea0c-cf9d-4906-9d85-3e9090c6e787)

I believe you can init them with standalone priv function, something like this: 
```rust
fn init_trade_fee_params() -> (u32, Vec<u8>, Vec<u8>, H160) {
    let timelock = now_sec_u32();
    let secret_hash = vec![0; 20];
    let swap_id = qrc20_swap_id(timelock, &secret_hash);
    let receiver_addr = H160::default();
    (timelock, secret_hash, swap_id, receiver_addr)
}
```

Reply:
well, I guess this is not this PR code.
(btw those parts are not totally same)